### PR TITLE
PR impact analysis coming soon

### PR DESCRIPTION
**Pull request by ellakz**

### CODE REVIEW

File: docs/cloud/general/ci.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+When making changes to your data project, it can sometimes be hard to fully understand the impact of your changes and be certain that there are no unintended consequences.
+
+Our impact analysis will run on every pull request in your dbt project, so that you can see the downstream impact of your changes.
+You'll also be able to see if any of your dbt tests are failing or your models aren't being built successfully.
+
+<img src="/pics/cloud/pr_impact_example.png" />
+
+Elementary CI automations will help you make changes with confidence and seeing the full picture before merging your pull request.
+
+## Want to learn more?
```
Comment (Maayan-s) [inline docs/cloud/general/ci.mdx:17]: Change to "Want to join the beta?"
And change the call to schedule a call or reach out to us on Slack

File: docs/mint.json [hunk 2]
```diff
@@ -224,7 +224,11 @@
     },
     {
       "group": "Introduction",
-      "pages": ["cloud/introduction", "cloud/general/security-and-privacy"]
+      "pages": [
+        "cloud/introduction",
+        "cloud/general/security-and-privacy",
+        "cloud/general/ci"
```
Comment (Maayan-s) [inline docs/mint.json:230]: Maybe create a new headline of "features"? seems strange here

File: docs/pics/cloud/pr_impact_example.png
Comment (Maayan-s) [inline docs/pics/cloud/pr_impact_example.png:1]: Looks amazing!

**Review by Maayan-s — CHANGES_REQUESTED** [review]
Awesome! Left some comments

File: docs/mint.json (hunk 2 above)
Comment (Maayan-s) [reply docs/mint.json:233]: @ellakz 
what do you think about this?

File: docs/cloud/guides/troubleshoot.mdx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Troubleshooting"
+---
+
+## Columns could not be loaded
+
+This error can happen because of a few reasons:
+
+1. check that your elementary dbt package version is 0.12.0 or higher
```
Comment (Maayan-s) [inline docs/cloud/guides/troubleshoot.mdx:9]: @ellakz 
0.12.0?

Comment (NoyaArie) [reply docs/cloud/guides/troubleshoot.mdx:9]: @haritamar I think you added a fallback for old versions, right?

**Review by Maayan-s — APPROVED** [review]
LGTM

**Pull request merged**